sdk/communication/azure-communication-networktraversal/src/main/java/com/azure/communication/networktraversal/models/RouteType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.networktraversal.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for RouteType. */
+public final class RouteType extends ExpandableStringEnum<RouteType> {
```

Comment:
The `RouteType` in implementation package is the same as this one. We should have just one of these.

Reply:
I'll create a work item to track this and fix in a separate PR.